platform/logging/appenders/base/BaseAppenderConfig.ts

```diff
@@ -0,0 +1,27 @@
+import * as schema from '../../../lib/schema';
+import { typeOfSchema } from "../../../types";
+import { BaseAppender } from './BaseAppender';
+
+export function getRawSchema() {
```

Comment:
These actually don't need to be functions. The schema can just be specified in a `const` instead.

(It makes sense to have it as a function if we inject `schema`, but we don't here, and I'm not sure we'll ever expose this stuff to plugins either)

Then you can use `schema.TypeOf` to get the type (just search for `TypeOf` to find examples)